insights/specs/insights_archive.py

```diff
@@ -234,6 +234,8 @@ class InsightsArchiveSpecs(Specs):
     systemctl_qdrouterd = simple_file("insights_commands/systemctl_show_qdrouterd")
     systemctl_smartpdc = simple_file("insights_commands/systemctl_show_smart_proxy_dynflow_core")
     systool_b_scsi_v = simple_file("insights_commands/systool_-b_scsi_-v")
+    teamdctl_config_dump = glob_file("insights_commands/usr/bin/teamdctl_*_config_dump")
+    teamdctl_state_dump = glob_file("insights_commands/usr/bin/teamdctl_*_state_dump")
```

Comment:
@rasrivas-redhat you need to remove the root path, `/usr/bin` from these two specs, it is not included in the mangled archive spec.

Reply:
@bfahr - I have updated the code, by mistakenly it happened.